config/ember-try.js

```diff
@@ -16,10 +18,20 @@ module.exports = {
       }
     },
     {
+      name: 'npm-test-loader',
+      npm: {
+        devDependencies: {
+          'ember-cli': 'trentmwillis/ember-cli#test-loader',
```

Comment:
this will need to obviously be updated before we can proceed (buy ya, abit chicken + egg)